### PR TITLE
manifest: Update tags for NCS v2.9.0-rc2 

### PR DESCRIPTION
**Pull request by jukkar**

### CODE REVIEW

**Review by shanthanordic — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Review by bjarki-andreasen — APPROVED** [review]

**Review by tejlmand — CHANGES_REQUESTED** [review]
should this be updated ?
https://github.com/nrfconnect/sdk-nrf/blob/a71d0c712723f31dc24bfdd2be7e0525627fd6f8/scripts/ncs-toolchain-version-minimum.txt#L3

**Comment by jukkar** [comment]
> should this be updated ?
> 
> https://github.com/nrfconnect/sdk-nrf/blob/a71d0c712723f31dc24bfdd2be7e0525627fd6f8/scripts/ncs-toolchain-version-minimum.txt#L3

No idea, tell me what to put there and I can send new version.

**Comment by jangalda-nsc** [comment]
> > should this be updated ?
> > https://github.com/nrfconnect/sdk-nrf/blob/a71d0c712723f31dc24bfdd2be7e0525627fd6f8/scripts/ncs-toolchain-version-minimum.txt#L3
> 
> No idea, tell me what to put there and I can send new version.

Please change to
2.8.20241119

**Review by tejlmand — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -206,12 +206,12 @@ manifest:
...
     - name: sidewalk
       repo-path: sdk-sidewalk
-      revision: v2.9.0-rc1
+      revision: v2.9.0-rc2
       groups:
         - sidewalk
     - name: find-my
       repo-path: sdk-find-my
-      revision: 0170d0374ffe379522a0d2eaf5b44d7d1aa1e7b7
+      revision: v2.9.0-rc2
```
Comment (kapi-no) [inline west.yml:214]: Incorrect tag. The v2.9-branch should have been used for tagging in the sdk-find-my repository

Comment (kapi-no) [reply west.yml:214]: I fixed this by force-pushing the tag aliases in the `sdk-find-my` repository.

**Pull request merged**